ethexe/observer/src/lib.rs

```diff
@@ -133,7 +133,7 @@ impl Stream for ObserverService {
             if let Some(header) = self.block_sync_queue.pop_front() {
                 let sync = ChainSync {
                     provider: self.provider.clone(),
-                    database: self.database.clone_boxed(),
+                    database: self.database.clone(),
```

Comment:
is clone_boxed about to be deprecated soon?

Reply:
If we would decide to do this) actually we use it inside db:
```
    pub fn from_one<DB: CASDatabase + KVDatabase>(db: &DB, router_address: [u8; 20]) -> Self {
        Self {
            cas: CASDatabase::clone_boxed(db),
            kv: KVDatabase::clone_boxed_kv(db),
            router_address,
        }
    }
```

Reply:
Renamed clone_boxed_kv -> clone_boxed

Reply:
Clone_boxed currently cannot be removed so leave it as is